src/js/edu-benefits/pages/applicantInformation.js

```diff
@@ -51,7 +51,14 @@ export default function applicantInformation(schema, options) {
       'ui:order': fields,
       relativeFullName: fullName.uiSchema,
       relativeSocialSecurityNumber: ssn.uiSchema,
-      relativeDateOfBirth: currentOrPastDate.uiSchema('Date of birth'),
+      relativeDateOfBirth: _.assign(currentOrPastDate.uiSchema('Date of birth'),
+        {
+          'ui:errorMessages': {
+            pattern: 'Please provide a valid date',
+            futureDate: 'Please provide a valid date'
```

Comment:
Having two separate ones makes sense to me, but I don't know why they're the same message or why this date field is special. If there's better generic copy, we should make that the default.

If we actually just want them to be the same message, let's just do that in the common date definition's uiSchema.

I think this depends on what @mollyblake is looking for here.